spring-pulsar/src/main/java/org/springframework/pulsar/listener/PulsarContainerProperties.java

```diff
@@ -335,4 +303,71 @@ private <T> void applyPropIfSpecified(String key, Consumer<T> setter) {
 		}
 	}
 
+	/**
+	 * Transaction related settings.
+	 *
+	 * @since 1.1.0
+	 */
+	public static class TransactionSettings extends TransactionProperties {
```

Comment:
These need sanity tests added - especially the `determineTransactionDefinition` method.